src/MailChimp/Facade.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace Locomotive\MailChimp;
+
+use ReflectionClass;
+use Mailchimp;
+
+/**
+ * File: MailChimp Facade Class
+ *
+ * Acts as a master facade class for MailChimp to allow me to isolate and test
+ * functionality without bootstrapping the vendor API.
+ *
+ * @package Locomotive\MailChimp
+ * @link    https://bitbucket.org/mailchimp/mailchimp-api-php/ Mailchimp API Client
+ * @see     Locomotive\WordPress\Facade {
+ *    Inspired by a modified version of Kevin Sperrine's WordPress facade.
+ * }
+ */
+
+/**
+ * Class: MailChimp Facade
+ *
+ * @version 2015-02-12
+ * @since   0.0.0 (2015-02-12)
+ */
+
+class Facade
+{
+	public  static $is_initialized = false;
+	private static $__facade;
```

Comment:
I think we should not put this property static. The possibility might happen where we need to deal with mulitple Mailchimp account in the future and that would prevent that possibility. A standard private instance should work just fine.